src/Mvc/test/Mvc.IntegrationTests/ValidationWithRecordIntegrationTests.cs

```diff
@@ -2298,7 +2298,9 @@ public async Task Validation_InifnitelyRecursiveModel_ValidationOnTopLevelParame
 
         private static void Validation_InifnitelyRecursiveModel_ValidationOnTopLevelParameterMethod([Required] RecursiveModel model) { }
 
+#pragma warning disable CS8907 // Parameter is unread. Did you forget to use it to initialize the property with that name?
```

Comment:
@pranavkm I'm assuming these records are made this way on purpose?

Reply:
Yup, intentional. This suppressions seems fine